apps/modernization-api/src/main/java/gov/cdc/nbs/graphql/filter/InvestigationFilter.java

```diff
@@ -32,23 +32,20 @@ public class InvestigationFilter {
     @Getter
     @Setter
     public static class CaseStatuses {
-        private boolean includeUnassigned;
         private List<CaseStatus> statusList;
     }
 
 
     @Getter
     @Setter
     public static class NotificationStatuses {
-        private boolean includeUnassigned;
         private List<NotificationStatus> statusList;
     }
 
 
     @Getter
     @Setter
     public static class ProcessingStatuses {
-        private boolean includeUnassigned;
         private List<ProcessingStatus> statusList;
     }
```

Comment:
Instead of having an Object to contain the Status list, we can move these Lists into the base InvestigationFilter